src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1336,11 +1336,15 @@ def update_site_configs(cmd, resource_group_name, name, slot=None, number_of_wor
                         vnet_route_all_enabled=None,
                         generic_configurations=None):
     configs = get_site_configs(cmd, resource_group_name, name, slot)
+    app_settings = _generic_site_operation(cmd.cli_ctx, resource_group_name, name,
+                                           'list_application_settings', slot)
```

Comment:
Since a new REST request `list_application_settings` are introduced here, please re-record those failed tests in CI pipeline

Reply:
Please re-record all test where this UpdateSiteConfig could be called.

Reply:
Sure Sisira.